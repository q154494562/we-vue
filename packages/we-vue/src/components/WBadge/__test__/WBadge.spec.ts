import { shallowMount } from '@vue/test-utils'
import Badge from '../'

describe('badge', () => {
  test('create', () => {
    const wrapper = shallowMount(Badge, {
      propsData: {},
    })

    expect(wrapper.classes()).toContain('weui-badge')
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('is-dot', () => {
    const wrapper = shallowMount(Badge, {
      propsData: {
        isDot: true,
      },
    })

    expect(wrapper.classes()).toContain('weui-badge_dot')
    expect(wrapper.html()).toMatchSnapshot()
  })
})
