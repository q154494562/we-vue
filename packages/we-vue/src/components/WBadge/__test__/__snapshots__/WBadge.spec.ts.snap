// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`badge create 1`] = `<span class="weui-badge"></span>`;

exports[`badge is-dot 1`] = `<span class="weui-badge weui-badge_dot"></span>`;
